builders/interact/ethereum-api/wallets/metamask.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to Connect MetaMask
+description: This guide walks you through how to connect MetaMask, a browser-based Ethereum wallet, to your Tanssi EVM ContainerChain and how to transfer funds.
+---
+
+# Interacting with your Tanssi EVM ContainerChain Using MetaMask
+
+## Introduction {: #introduction } 
+
+Developers building dApps on top of Tanssi EVM ContainerChains can leverage their Ethereum compatibility features integrate known Ethereum wallets, such as [MetaMask](https://metamask.io/){target=_blank}. By doing so, they can use the injected library MetaMask provides to interact with the Tanssi EVM ContainerChain.
+
+This guide takes you through all the necessary steps: from installing Metamask, to setting up a wallet, and finally connecting it to your Tanssi EVM ContainerChain.
+
+## Install the MetaMask Extension {: #install-the-metamask-extension } 
+
+First, you'll start with a fresh and default [MetaMask](https://metamask.io/){target=_blank} installation from the Chrome store. After downloading, installing, and initializing the extension, follow the **Get Started** guide. In there, you need to create a wallet, set a password, and store your secret backup phrase (this gives direct access to your funds, so make sure to store these in a secure place). 
```

Comment:
```suggestion
First, you'll start with a fresh and default [MetaMask](https://metamask.io/){target=_blank} installation from the Chrome store. After downloading, installing, and initializing the extension, follow the **Get Started** guide it will present. In there, you need to create a wallet, set a password, and store your secret backup phrase (this gives direct access to your funds, so make sure to store these in a secure place). 
```